### PR TITLE
Create results and homepage

### PR DESCRIPTION
**Pull request by srirag-vuppala**
Tasks Achieved
---
- Create results component
image

- Set up homepage
image

- Allow filtering through dropdown and search bar
- Searchbar image
- Filter Dropdown
image

- Added emojis to make everything just pop out a bit more (Additional feature)
image

### CODE REVIEW

**Review by hhokari — APPROVED** [review]
Looks good!

**Comment by hhokari** [comment]
I wanted to also add that the website looks good overall, but the responsiveness of the website on different screen sizes (half, quarter size) needs some improvement. Moreover, the text and components overlap with each other.

**Comment by srirag-vuppala** [comment]
True! I feel like responsiveness would be really great but given our time frame I don't think it would fit in our MVP

**Comment by hhokari** [comment]
> True! I feel like responsiveness would be really great but given our time frame I don't think it would fit in our MVP

**Review by hhokari — COMMENTED** [review]
react_frontend/src/components/Home.js
I also wanted to ask what is the react select in Home.js? I also wanted to address that you should try to remove unnecessary imports that you do not need.

File: react_frontend/src/components/Home.js [hunk 1]
```diff
@@ -1,20 +1,85 @@
-import React from 'react'
-import '../assets/NavBar.css'
+import React, { useState, useEffect } from 'react'
+import '../assets/Home.css'
 import SearchBar from './SearchBar'
 import SearchButton from './SearchButton'
```
Comment (hhokari) [inline react_frontend/src/components/Home.js:4]: I also wanted to ask what is the react select in Home.js? I also wanted to address that you should try to remove unnecessary imports that you do not need.

**Review by hhokari — COMMENTED** [review]
.

File: react_frontend/src/components/Home.js (hunk 1 above)
Comment (srirag-vuppala) [reply react_frontend/src/components/Home.js:4]: Good catch! I'll go ahead and fix that 👍🏾

**Comment by hhokari** [comment]
> True! I feel like responsiveness would be really great but given our time frame I don't think it would fit in our MVP

One other thi

**Comment by srirag-vuppala** [comment]
opening it back up for a merge

**Pull request merged**